services/types.go

```diff
@@ -1,6 +1,7 @@
 package services
 
 type OnboardingTicket struct {
-	ID             string `json:"id"`
-	ExpirationDate int64  `json:"expirationDate,string"`
+	ID                string `json:"id"`
+	ExpirationDate    int64  `json:"expirationDate,string"`
+	StorageQuotaInGiB uint   `json:"storageQuotaInGiB,omitempty"`
```

Comment:
Once this pull request (https://github.com/red-hat-storage/ocs-operator/pull/2600/files) is merged, I will rebase this. Then there won't be any changes in the structure.